service/topology_coordinator.cc

```diff
@@ -2485,18 +2485,17 @@ future<bool> topology_coordinator::maybe_start_tablet_split_finalization(group0_
 
 future<locator::load_stats> topology_coordinator::refresh_tablet_load_stats() {
     auto tm = get_token_metadata_ptr();
-    auto& topology = tm->get_topology();
 
     locator::load_stats stats;
     static constexpr std::chrono::seconds wait_for_live_nodes_timeout{30};
 
     std::unordered_map<table_id, size_t> total_replicas;
 
-    for (auto& [dc, nodes] : topology.get_datacenter_nodes()) {
+    for (auto& [dc, endpoints] : tm->get_datacenter_token_owners()) {
```

Comment:
This is another place where we replace node-based code with IP-based code

Reply:
I fixed that with new `get_datacenter_token_owners_nodes`.